#!/usr/bin/env python
PACKAGE = "project_1"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("threshold_1_param", double_t, 0, "parameter for crash threshold",    1, 0,   50)
gen.add("threshold_2_param", double_t, 1, "parameter for unsafe threshold",    5, 0,   50)

gen.generate(PACKAGE, "dyna_config_node", "dynaParam")
